functests/11-test-node-labeller-placement.sh

```diff
@@ -55,6 +55,31 @@ if [ "$AFFINITY" != "testKey" ]; then
   RET=1
 fi
 
+oc apply -n ${TEST_NS} -f "${RES_DIR}/11-node-labeller-empty-affinity-nodeSelector-tolerations.yaml" || exit 2
+#wait until operator applies newest configuration
+sleep 60
+
+NODE_SELECTOR=$(oc get -n ${TEST_NS} ds kubevirt-node-labeller -ojson | jq '.spec.template.spec.nodeSelector.testKey' | tr -d '"')
+if [ "$NODE_SELECTOR" == "testValue" ]; then
```

Comment:
Where the YAML applied empties the spec values, we may want to check the result for a non-empty condition instead. If the entry is modified, but not deleted, that would still be a failure.

This would also apply to the two tests following this one.